analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/metrics/AgeInWeeks.kt

```diff
@@ -0,0 +1,26 @@
+package de.maibornwolff.codecharta.importer.scmlogparser.input.metrics
+
+import de.maibornwolff.codecharta.importer.scmlogparser.input.Commit
+import java.time.OffsetDateTime
+
+class AgeInWeeks : Metric {
+    private var firstCommit: CalendarWeek = CalendarWeek.forDateTime(OffsetDateTime.now())
```

Comment:
Do we need the "today's week" default value?

Reply:
well, if there is always a commit present for each VersionControlledFile, then no. However if for some reason we do not have a commit, we would get a null value. Whether that case will occur in reality, i cannot answer definitely, however it would mean that we would have to ensure null safety somewhere else, and i think having a sensible default value is better in this case.

Reply:
yea makes sense, and I also checked what other metrics do by default. They all set a default value. So it seems to be fine 👍